ppm_telegram_bot/dependencies.py

```diff
@@ -17,3 +23,7 @@ def bot_dispatcher() -> Dispatcher:
     Bot.set_current(dispatcher.bot)
     Dispatcher.set_current(dispatcher)
     return dispatcher
```

Comment:
It seems that here we call Bot's and Dispatcher's `set_current` on every API call. Is this behaviour expected, or should we maybe cache this function's result?

Reply:
Expected, we need to do it